src/database/index.js

```diff
@@ -5,6 +5,11 @@ const {
 } = process.env
 
 if (NODE_ENV === 'test' || NODE_ENV === 'development') {
+  dynamoose.aws.sdk.config.update({
+    accessKeyId: 'AKID',
+    secretAccessKey: 'SECRET',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
    secretAccessKey: "SECRET",
```